app/helpers/actual_db_schema/migrations_helper.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module ActualDbSchema
+  # Helper methods for loading and displaying migrations.
+  module MigrationsHelper
+    def load_migrations
```

Comment:
By helper method I meant this https://stackoverflow.com/a/3993323

You can find a lot of examples using it inside BudgetingKid